arbitrator/wasm-testsuite/check.sh

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022, Offchain Labs, Inc.
```

Comment:
Let's put a `#!/usr/bin/bash` at the top

Reply:
Added